Meta/tweet-commits.js

```diff
@@ -13,11 +16,13 @@ const T = new Twit({
     const githubEvent = JSON.parse(fs.readFileSync(0).toString());
     const tweets = [];
     for (const commit of githubEvent["commits"]) {
-        tweets.push(
-            `${commit["message"].substring(0, 240)}\nAuthor: ${commit["author"]["name"]}\n${
-                commit["url"]
-            }`
-        );
+        const authorLine = `Author: ${commit["author"]["name"]}`;
+        const maxMessageLength = tweetLength - authorLine.length - twitterLinkLength - 2;
```

Comment:
Where does the `2` come from? You have two newlines around the `${authorLine}` but the commit message itself contains breaks as well to separate the commit summary and description.